CONTRIBUTING.md

```diff
@@ -12,3 +12,23 @@ The base version represents the MAJOR and MINOR parts of [SemVer](https://semver
 
 - `main` contains the latest sources. Each merge there triggers a deploy to `nuget.org`.
 - All versions on `nuget.org` have a matching GitHub release/tag.
+
+## Commits and PR title
+
+- Please use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) to name your commits and PR title
+  Available types:
```

Comment:
I think "available prefixes" would be more clear.

Reply:
sure thing. Fixed